Tools/autotest/sim_vehicle.py

```diff
@@ -698,6 +698,17 @@ def start_mavproxy(opts, stuff):
     run_cmd_blocking("Run MavProxy", cmd, env=env)
     progress("MAVProxy exitted")
 
+def mkdir_p(dir):
+    '''like mkdir -p'''
+    if not dir:
+        return
+    if dir.endswith("/"):
+        mkdir_p(dir[:-1])
+        return
```

Comment:
I'm not saying removing the if, but I see no reason to do a recursive call here.